Proton/ruleset.xml

```diff
@@ -74,6 +74,9 @@
                 <element key="intval" value="(int)"/>
                 <element key="floatval" value="(float)"/>
                 <element key="boolval" value="(bool)"/>
+
+                <!-- Use namepaced function instead of global -->
+                <element key="config" value="\Proton\Support\config"/>
```

Comment:
I prefer to keep this loosely coupled to our codebase, we can add this in each internal project

Reply:
I removed it